nixos/modules/services/hardware/openrgb.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+  cfg = config.services.hardware.openrgb;
+in {
+  options.services.hardware.openrgb = {
+    enable = mkEnableOption "OpenRGB server";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.openrgb;
+      defaultText = "pkgs.openrgb";
+      description = "Set version of openrgb package to use.";
+    };
+
+    motherboard = mkOption {
+      type = types.nullOr (types.enum [ "amd" "intel" ]);
+      default = null;
+      defaultText = "null";
```

Comment:
defaultText for literals isn't needed, and i'd argue it makes things less maintainable. (even that defaultText *always* makes things less maintainable, but sometimes it's necessary)